diff --git a/testsuite/E10.py b/testsuite/E10.py
deleted file mode 100644
index 9ec1d41..0000000
--- a/testsuite/E10.py
+++ /dev/null
@@ -1,8 +0,0 @@
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
diff --git a/testsuite/E11.py b/testsuite/E11.py
deleted file mode 100644
index 0db0de1..0000000
--- a/testsuite/E11.py
+++ /dev/null
@@ -1,16 +0,0 @@
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
diff --git a/testsuite/E12.py b/testsuite/E12.py
deleted file mode 100644
index b53c210..0000000
--- a/testsuite/E12.py
+++ /dev/null
@@ -1,5 +0,0 @@
-remove me all
-remove me all
-remove me all
-remove me all
-remove me all
