diff --git a/testsuite/E10.py a/testsuite/E10.py
new file mode 100644
index 9ec1d41..0000000
+++ b/testsuite/E10.py
--- /dev/null
@@ -1,8 +2,9 @@
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
diff --git a/testsuite/E11.py a/testsuite/E11.py
new file mode 100644
index 0db0de1..0000000
+++ b/testsuite/E11.py
--- /dev/null
@@ -1,16 +2,17 @@
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
+add me
diff --git a/testsuite/E12.py a/testsuite/E12.py
new file mode 100644
index b53c210..0000000
+++ b/testsuite/E12.py
--- /dev/null
@@ -1,5 +2,6 @@
+add me
+add me
+add me
+add me
+add me
