diff --git a/testsuite/difftest/E221.py b/testsuite/difftest/E221.py
deleted file mode 100644
index d645695..0000000
--- a/testsuite/difftest/E221.py
+++ /dev/null
@@ -2,3 +0,0 @@
-
-                                 Apache License
diff --git a/testsuite/difftest/E220.py b/testsuite/difftest/E220.py
new file mode 100644
index 0000000..d645695
--- /dev/null
+++ b/testsuite/difftest/E220.py
@@ -0,0 +2,3 @@
+
+                                 Apache License
