diff --git a/testsuite/difftest/Diff_test_1.py b/testsuite/difftest/Diff_test_1.py
new file mode 100644
index 0000000..f3d341b
--- a/testsuite/difftest/Diff_test_1.py
+++ b/testsuite/difftest/Diff_test_1.py
@@ -4,6 +4,7 @@
I am here
I am here
I am here
+add me

I am here

@@ -11,8 +12,8 @@
I am here


-remove me
-remove me
+add me
+add me


 I am here
@@ -28,14 +29,6 @@
I am here
I am here

-remove me
-remove me
-remove me
-remove me
-remove me
-remove me
-remove me
-remove me
I am here
I am here

@@ -36,8 +36,17 @@
I am here


+add me
I am here


I am here
I am here
+add me
